custom/enikshay/nikshay_datamigration/models.py

```diff
@@ -286,8 +286,8 @@ class Outcome(models.Model):
     @property
     def hiv_status(self):
         return {
-            None: None,
-            'NULL': None,
+            None: 'unknown',
```

Comment:
When this is set to unknown or 'reactive', we need to make a drtb-hiv-referral case to the drtb-hiv location under the dto location.